### PR TITLE
[Pager] Move indicators to pager-indicators module

### PR DESCRIPTION
**Pull request by chrisbanes**
This allows us to make the indicator module depend on Compose Material, and provide better default values for the indicators. Also simplified the parameters to make them more consistent with other Compose layouts.

### CODE REVIEW

File: pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+ */
+@ExperimentalPagerApi
+@Composable
+fun HorizontalPagerIndicator(
+    pagerState: PagerState,
+    modifier: Modifier = Modifier,
+    activeColor: Color = LocalContentColor.current.copy(alpha = LocalContentAlpha.current),
+    inactiveColor: Color = activeColor.copy(ContentAlpha.disabled),
+    indicatorWidth: Dp = 8.dp,
+    indicatorHeight: Dp = 8.dp,
```
Comment (CaptnBlubber) [inline pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt:66]: Maybe use `indicatorWidth` here as default value so its symetric by default?

**Review by CaptnBlubber — COMMENTED** [review]
Looks good to me thanks for migrating it to a seperate Module and applying the default color changes

File: pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt (hunk 1 above)
Comment (chrisbanes) [reply pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt]: Good call, done.

**Comment by FilippoVigani** [comment]
Sorry to commend on an old issue, but I think it would make sense to call the library `accompanist-pager-indicators-material` since it depends on Compose Material. The same way the placeholder has the library `accompanist-placeholder-material`.

**Pull request merged**